src/containers/Header/BreadcrumbsRouter/Breadcrumbs.js

```diff
@@ -36,10 +36,9 @@ const Breadcrumbs = ({
 	useEffect(() => {
 		// Adding subtitle to the title of the application
 		if (app.Services.TitleService != undefined) {
-			app.Services.TitleService.setTitle(crumbs[0].name);
+			app.Services.TitleService.setTitle(t(`Breadcrumbs|${crumbs[0].name}`));
 		}
-	}, [crumbs]);
-
+	}, [crumbs, t]);
```

Comment:
@Aringoaway Oh, you dont need `t` here. Why it has been added?

Reply:
And since it has been already refactored, I suppose this PR can be closed:

https://github.com/TeskaLabs/asab-webui/blob/master/src/containers/Header/BreadcrumbsRouter/Breadcrumbs.js#L41

Reply:
@Pe5h4 I added `t` because I felt that when you change the language, it should change in the title as well

This video shows your changes without `t`
https://github.com/TeskaLabs/asab-webui/assets/65866093/afbcfda1-6b06-47b0-b545-9ac7cc48cd93

Reply:
@Aringoaway Ok, you are right